contributing-device-support.md

```diff
@@ -0,0 +1,66 @@
+# Contributing new board support to resin.io
+
+Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible to yocto daisy/fido/jethro release at the time of writing this howto.
+
+Every repository used to build a resin.io host Operating System is tipically named resin-`<board-family>`. For example, consider the resin-beaglebone repository which can used to build a resin.io image for
+the BeagleBone Black, BeagleBone Green and BeagleBone Green WiFi boards.
+
+### Repository breakout
+
+The resin-`<board-family>` repository uses git submodules for including required subprojects.
+
+The root directory shall contain 3 directory entries: a "layers" directory and the resin-device-types (https://github.com/resin-io/resin-device-types) and
+resin-yocto-scripts (https://github.com/resin-os/resin-yocto-scripts) git submodules. These submodules are tipically using latest master branches of the
+respective git repositories.
+The root directory also includes the following files: CHANGELOG.md, LICENSE, README.md, VERSION and one or more files (depending on the number of supported boards)
+named `<yocto-machine-name>`.coffee which correspond to each of the boards this repository will add support for.
```

Comment:
It is the files that correspond. I thought it's plural here that needs to be used